setup.cfg

```
@@ -47,9 +47,7 @@ testing =
 	pytest-cov; \
 		# coverage seems to make PyPy extremely slow
 		python_implementation != "PyPy"
-	pytest-mypy >= 0.9.1; \
-		# workaround for jaraco/skeleton#22
-		python_implementation != "PyPy"
+	pytest-mypy
```

https://github.com/jaraco/skeleton/pull/111

---

Should we wait to see how that issue goes.

Even if we don't change this dependency, I believe the tests will still run on the CI (for everything else that is not PyPy).

---

I don't think this change is necessary anymore since I'm no longer unconditionally adding the `--mypy` flag. As you said, it can be brought back from upstream later.
```suggestion
	pytest-mypy >= 0.9.1; \
		# workaround for https://github.com/jaraco/skeleton/issues/22
		python_implementation != "PyPy"
```